xcodeproj/code_sign_info.go

```diff
@@ -58,7 +58,6 @@ func readSchemeTargetMapping(projectPth, scheme, user string) (TargetMapping, er
 		return TargetMapping{}, fmt.Errorf("failed to run ruby script, output: %s, error: %s", out, err)
```

Comment:
please write an error message describing what we've lost by being unable to run the ruby script